modules/motions/ui/MotionFormStartNew/Parts/StartNewNodeOperatorLimitIncrease.tsx

```diff
@@ -0,0 +1,154 @@
+      const { setValue } = useFormContext()
+      const { walletAddress } = useWeb3()
+      const registryType = getNodeOperatorRegistryType(motionType)
+      const nodeOperators = useNodeOperatorsList(registryType)
+      const keysInfo = useNodeOperatorKeysInfo(registryType)
+
+      const currentNodeOperator = useMemo(() => {
+        if (!walletAddress || !nodeOperators.data) {
+          return null
+        }
+        return nodeOperators.data.find(
+          o =>
+            utils.getAddress(o.rewardAddress) ===
+            utils.getAddress(walletAddress),
+        )
+      }, [nodeOperators.data, walletAddress])
+
+      const isNodeOperatorConnected =
+        currentNodeOperator && walletAddress && nodeOperators.data
+
+      const connectedKeysInfo = useMemo(() => {
+        if (!isNodeOperatorConnected || !keysInfo.data || !walletAddress)
+          return null
+        return keysInfo.data.operators?.find(
+          o =>
+            utils.getAddress(o.info.rewardAddress) ===
+            utils.getAddress(walletAddress),
+        )
+      }, [isNodeOperatorConnected, keysInfo.data, walletAddress])
+
+      useEffect(() => {
+        if (currentNodeOperator) {
+          setValue(fieldNames.nodeOperatorId, currentNodeOperator.id)
+        }
+      }, [fieldNames.nodeOperatorId, setValue, currentNodeOperator])
+
+      if (nodeOperators.initialLoading || keysInfo.initialLoading) {
+        return <PageLoader />
+      }
+
+      if (!isNodeOperatorConnected) {
+        return <MessageBox>You should be connected as node operator</MessageBox>
+      }
+
+      if (!connectedKeysInfo) {
+        return <MessageBox>Error: No keys info found</MessageBox>
+      }
+
+      const isConnectedKeysValid =
+        connectedKeysInfo.invalid.length === 0 &&
+        connectedKeysInfo.duplicates.length === 0
+
+      if (!isConnectedKeysValid) {
+        return <KeysInfoBlock keys={connectedKeysInfo} />
+      }
+
+      return (
+        <>
+          <KeysInfoBlock keys={connectedKeysInfo} />
+
+          <Fieldset>
+            <InputControl
+              name={fieldNames.nodeOperatorId}
+              label={
+                <>
+                  Node operator <b>{currentNodeOperator.name}</b> with id
+                </>
+              }
+              rules={{ required: 'Field is required' }}
+              readOnly
+            />
+          </Fieldset>
+
+          <Fieldset>
+            <InputNumberControl
+              name={fieldNames.newLimit}
+              label={
+                <>
+                  New limit (current limit is{' '}
+                  {currentNodeOperator.stakingLimit.toString()})
+                </>
+              }
+              rules={{
+                required: 'Field is required',
+                validate: value => {
+                  if (value === '') return true
+                  const parsedValue = Number(value)
+                  if (Number.isNaN(parsedValue)) return 'Wrong number format'
```

Comment:
You could use `isNaN` here instead of `Number.isNaN`. I will work same, because you used `Number(value)` at previous line.